### ISSUE TITLE
Localstack start - duplicate -p 8080:8080, results in error

### ISSUE THREAD

**Issue by arielweinberger** [localstack/localstack#2272]
# Type of request: This is a ...

[x] bug report
[ ] feature request

# Detailed description

Localstack docker run command contains `-p 8080:8080` twice, which results in an error. When running the following command manually, I can successfully run Localstack:

```
docker run -it -e LOCALSTACK_HOSTNAME="localhost" -e TEST_AWS_ACCOUNT_ID="000000000000" -e DEFAULT_REGION="us-east-1" \
-p 443:443 \
-p 4566:4566 \
-p 8081:8081 \
-p 4567-4617:4567-4617 \
-p 8080:8080  \
--rm --privileged --name localstack_main \
-v "/private/var/folders/01/wnz8g_c95fx81w85tpl2dx300000gn/T/localstack:/tmp/localstack" -v "/var/run/docker.sock:/var/run/docker.sock" -e DOCKER_HOST="unix:///var/run/docker.sock" -e HOST_TMP_FOLDER="/private/var/folders/01/wnz8g_c95fx81w85tpl2dx300000gn/T/localstack" "localstack/localstack"
```

**OS:** MacOS Mojave 10.14.6
**Docker version:** Docker version 19.03.8, build afacb8b

## Expected behavior

Running `localstack start` should set up the stack as expected.

## Actual behavior

Running `localstack start` results in the following Docker error:
```
docker: Error response from daemon: Ports are not available: /forwards/expose/port returned unexpected status: 500.
ERRO[0000] error waiting for container: context canceled
```

# Steps to reproduce
1. Install Localstack - `pip install localstack`
2. Run Localstack - `localstack start`
3. Error is displayed

## Command used to start LocalStack
`localstack start`

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)
N/A

**Comment by rpehrson-takeda** [comment]
Same issue, confirm resolution on my end as well.

**Comment by whummer** [comment]
Thanks for reporting @arielweinberger @rpehrson-takeda . This should be fixed in #2280 . A patched version has been pushed to pypi - can you please try upgrading to the latest version:
```
pip install --upgrade --no-cache localstack
```
Please let us know if the problem persists. Thanks

**Comment by pzrq** [comment]
Curiously this just happened to me upgrading localstack from 0.11.2 to 0.12.2.

Got past it by deleting localstack from the `Containers / Apps` area of the macOS Docker Dashboard and rerunning the command line. 
<img width="327" alt="Screen Shot 2020-11-13 at 4 03 44 pm" src="https://user-images.githubusercontent.com/1217010/99031030-26d44880-25ca-11eb-9a68-d8a3adced2e1.png">
